capa/features/insn.py

```diff
@@ -15,9 +15,9 @@
 def hex(n: int) -> str:
     """render the given number using upper case hex, like: 0x123ABC"""
     if n < 0:
-        return "-0x%X" % (-n)
+        return f"-0x{(-n):X}"
     else:
-        return "0x%X" % n
+        return f"0x{(n):X}"
```

Comment:
are the `()` required here (for code style by black, for example)?
Otherwise, simplify to `f"-0x{-n:X}"` and below as well?

Reply:
That's a valid point. I think we wanted to have more control over it (e.g. to force it to be uppercase) and use it consistently so all numbers are formatted similarly. In this PR we can see that this isn't really working. We can create a new issue for it to simplify things here.

Reply:
Thanks for the  insight. As for the brackets, they are required by black.